editor-components/call_to_action/call_to_action7/call_to_action7.tsx

```diff
@@ -1,143 +1,105 @@
+      value: "https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/6687a7beba6bbe002b63bb11?alt=media",
     });
-    // this.addProp({
-    //   type: "color",
-    //   key: "cardBackgroundColor",
-    //   displayer: "Card Background Color",
-    //   value: "#e0e0e0",
-    // });
-
     this.addProp({
-      type: "array",
-      key: "item-list",
-      displayer: "Card List",
+      type: "object",
+      key: "button",
+      displayer: "Button",
       value: [
         {
-          type: "object",
-          key: "item",
-          displayer: "Item",
-          value: [
-            {
-              type: "string",
-              key: "Title",
-              displayer: "Title",
-              value: "About Life",
-            },
-            {
-              type: "string",
-              key: "Description",
-              displayer: "Description",
-              value:
-                "Life is a journey full of ups and downs, joys and sorrows, victories and defeats. It's a precious gift that should be cherished and lived to the fullest. Life is unpredictable and can change in an instant, so it's important to appreciate the present moment and make the most of every opportunity. ",
-            },
-            {
-              type: "string",
-              key: "ButtonText",
-              displayer: "Button Text",
-              value: "Reading",
-            },
-            {
-              type: "page",
-              key: "ButtonPage",
-              displayer: "Button Link",
-              value: "https://www.google.com/",
-            },
-            {
-              type: "image",
-              key: "ButtonIcon",
-              displayer: "Button Icon",
-              value:
-                "https://freepngimg.com/thumb/web_design/24713-2-right-arrow-transparent-background.png",
-            },
-            {
-              type: "image",
-              key: "card-icon",
-              displayer: "Card Icon",
-              value:
-                "https://upload.wikimedia.org/wikipedia/commons/thumb/a/ae/Globe_icon-white.svg/1200px-Globe_icon-white.svg.png",
-            },
-          ],
+          type: "string",
+          key: "buttonText",
+          displayer: "Button Text",
+          value: "LOOK MORE",
         },
-      ],
+        {
+          type: "page",
+          key: "link",
+          displayer: "Link",
+          value: "",
+        }
+      ]
     });
+    this.addProp({
+      type: "string",
+      key: "placeholder",
+      displayer: "Placeholder",
+      value: "Enter your E-Mail!",
+    });
+
   }
 
   getName(): string {
     return "Call To Action 7";
   }
 
   render() {
-    const styling = {
-      backgroundColor: this.getPropValue("cardBackgroundColor"),
-    };
+    const title = this.getPropValue("title");
+    const image = this.getPropValue("image");
+    const buttonProps = this.getPropValue("button");
+    const buttonText = buttonProps[0].value;
```

Comment:
Don't access any field like this. Please create a type for this prop.

[Please read this.](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md)